### ISSUE TITLE
Plugin not available for VScode

### ISSUE THREAD

**Issue by sanalkv** [zhangruiyu/FlutterJsonBeanFactory#54]
I couldn't find this plugin for vscode. Hope it's in development.

**Comment by vishalpatel1327** [comment]
yes because when not every developer is using android studio. that's a big blockage for us to force them to use android studio for that.
We humble request that we love to see same plugin in VScode
